main.py

```diff
@@ -0,0 +1,51 @@
+originArray = [[1,2,3],[4,5,6],[7,8,9]]
+finalArray = []
+
+# append to final array leftmost column of origin array
+def goUp(a):
```

Comment:
I suggest picking another name for the parameter. Can you think of a better name? :)